geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/wancommand/ListGatewaysCommandDUnitTest.java

```diff
@@ -369,4 +369,144 @@ public void listGatewaysShouldCorrectlyUpdateSendersConnectedCountWhenReceiverSt
         .hasRowSize(1)
         .hasColumn("Sender Count").containsExactly("0");
   }
+
+  @Test
+  public void testListGatewaySenderOnly() {
```

Comment:
Could the test names be changed slightly to include the expected behaviour as well as the conditions? For example, this test would change to "testListGatewaySenderOnlyListsOnlyGatewaySenders." It can feel redundant in cases where the expected behaviour should be obvious from the wording of the command, but in other cases, such as when both options are specified, it can help make the intent of the test clearer.